.env-example

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=
```

Comment:
hi @uthdev could you please check this line, I can't find its use

Reply:
Thank you @Oliver-ke, that's that is just for the production database URL. Put it there so we know you'll also need in the environment variables. We are going to be having 3 databases each for development, testing and production. Check out the config file in `/src/database/config/` directory to confirm this.